.github/workflows/docs-check.yml

```diff
@@ -43,9 +43,13 @@ jobs:
             !Documentation/Helm-Charts
 
       - name: Check helm-docs
-        run: make check.helm-docs
+        run: |
+          make gen.helm-docs
+          tests/scripts/validate_modified_files.sh helm-docs
       - name: Check docs
-        run: make check.docs
+        run: |
+          make gen.docs
```

Comment:
I wonder why we have both `make gen.docs` and `make gen.helm-docs`. They both just call the `helm-docs` action internally. Shall we remove one of them, or what's different?

Reply:
I was trying to follow the pattern `gen.foo`introduced earlier for generating targets.

Agreed, this introduces redudancy. Which would you like me to remove?

Reply:
How about if we keep all the targets as you've implemented, but remove one of them from docs-check.yml?
For example, keep the call to "Check docs", but remove "Chek helm-docs" since it's a redundant check

Reply:
done